### DISCUSSION TITLE
build our app in Docker，

### DISCUSSION THREAD

**Discussion by licongwen** [vikejs/vike#1747] [Discussion] [answered]
**we build our app in Docker, how do we startup a docker? i used 'vite preview', but it is not recommend in production. which script should i use?**

``` js
"scripts": { 
    "dev:uat": "cross-env NODE_ENV=development  PORT=80 npm run server:dev",
    "dev:prod": "cross-env NODE_ENV=development PORT=80 npm run server:prod",
    "build:uat": "cross-env NODE_ENV=production vite build",
    "build": "cross-env NODE_ENV=production vite build",
    "server": "node --loader ts-node/esm ./server/index.ts",
    "server:dev": "cross-env NODE_ENV=production  PORT=80 npm run server",
    "server:prod": "cross-env NODE_ENV=production  PORT=80 npm run server",
    "lint": "eslint . --maserx-warnings 0",
    "preview": "vite preview"
}
```
Otherwise，i used 'npm run server:prod' as docker startup script，but in production，i get some error：
![image](https://github.com/user-attachments/assets/b951c323-d4c0-4954-a193-6d6ca1fd3c28)
i am already set cross-env NODE_ENV=production, how can i solve with this errors?

**Comment by brillout** [comment, marked as answer]
Your issue doesn't seem related to Vike, but feel free to elaborate if you think it is. (Adding getting started docs to help with setting up an full-stack JavaScript architecture is on the radar.)
